### DISCUSSION TITLE
How do I add the Accessibility tag to an issue I've created?

### DISCUSSION THREAD

**Discussion by ChasBelov** [community/community#47921] [New to GitHub] [answered]
I have created several issues related to Accessibility on the Mastodon project. I see that Mastodon does have existing issues tagged for Accessibility. However, I don't see how to tag my new issues with the accessibility tag. Please advise.

**Comment by melvincwng** [comment, marked as answer]
@ChasBelov, to create custom labels (such as accessibility) and then apply/tag it to your issues or PR, please kindly refer to this GitHub Documentation link over here: https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels

Hope this clarifies and answers your question 😃 

Take care, and have a good day ahead.

**Reply by ChasBelov** [reply]
Aha, thank you, that documentation explains that only people with triage or write permission on an issue queue can add tags beyond bug or suggestion. I don't have those permissions as I'm not part of the project, so I don't get the button to add additional tags/labels.

**Reply by melvincwng** [reply]
Glad to have helped! 👍🏻
